src/index.js

```diff
@@ -59,6 +83,114 @@ document.addEventListener('DOMContentLoaded', () => {
 
 ///////////////////////////// Internal functions /////////////////////////////
 
+/**
+ * read json data produced by django's `json_script` filter during HTML template generation
+ */
+function readDjangoJsonScript(id) {
+  const jsonScriptElement = document.getElementById(id)
+  if (jsonScriptElement){
+    return JSON.parse(jsonScriptElement.textContent)
+  }else{
+    return undefined
+  }
```

Comment:
Okay, so this is really dumb, but the following is equivillent:

```suggestion
  }
```

JavaScript functions return `undefined` if there is no return statement.

Reply:
nice. I actually didn't know